thunderfish/sorting_tools.py

```diff
@@ -58,80 +60,120 @@ def wave_or_pulse_psd(power, freqs, data, rate, fresolution, create_dataset=Fals
+
     if np.mean(proportions) < 0.25:
         psd_type = 'wave'
     else:
         psd_type = 'pulse'
 
-    # if diff < 15 and np.mean(proportions) < 0.25:
-    #     psd_type = 'wave'
-    # elif diff > 15 and np.mean(proportions) > 0.25:
-    #     psd_type = 'pulse'
-    # else:
-    #     psd_type = 'unknown'
-
-    #####################################################################################
-    # Creating dataset #
+    ################################################################################################
+    # new idea: skewness
+    power_db = 10.0 * np.log10(power[:int(1500/fresolution)])
+    skewness = sps.kurtosis(power_db) ####
 
+    ################################################################################################
     if create_dataset is True:
-        plt.draw()
-        plt.pause(1)
-
-        response = raw_input('wave- or pulse psd ? [w/p]')
-        plt.close()
-        if response == "w":
-            file = "wave_diffs.npy"
-            file2 = "wave_proportions.npy"
-            file3 = "wave_trace_proportions.npy"
-        elif response == "p":
-            file = "pulse_diffs.npy"
-            file2 = "pulse_proportions.npy"
-            file3 = "pulse_trace_proportions.npy"
+        if kategory is 'wave':
+            if not os.path.exists('wave_PSD_algor.npy'):
+                np.save('wave_PSD_algor.npy', np.array([]))
+            wave_prop = np.load('wave_PSD_algor.npy')
+            wave_prop = wave_prop.tolist()
+            wave_prop.append(np.mean(proportions))
+            wave_prop = np.asarray(wave_prop)
+            np.save('wave_PSD_algor.npy', wave_prop)
+
+            if not os.path.exists('wave_skewness.npy'):
+                np.save('wave_skewness.npy', np.array([]))
+            wave_skewness = np.load('wave_skewness.npy')
+            wave_skewness = wave_skewness.tolist()
+            wave_skewness.append(skewness)
+            wave_skewness = np.asarray(wave_skewness)
+            np.save('wave_skewness.npy', wave_skewness)
+        elif kategory is 'pulse':
+            if not os.path.exists('pulse_PSD_algor.npy'):
+                np.save('pulse_PSD_algor.npy', np.array([]))
+            pulse_prop = np.load('pulse_PSD_algor.npy')
+            pulse_prop = pulse_prop.tolist()
+            pulse_prop.append(np.mean(proportions))
+            pulse_prop = np.asarray(pulse_prop)
+            np.save('pulse_PSD_algor.npy', pulse_prop)
+
+            if not os.path.exists('pulse_skewness.npy'):
+                np.save('pulse_skewness.npy', np.array([]))
+            pulse_skewness = np.load('pulse_skewness.npy')
+            pulse_skewness = pulse_skewness.tolist()
+            pulse_skewness.append(skewness)
+            wave_skewness = np.asarray(pulse_skewness)
+            np.save('pulse_skewness.npy', pulse_skewness)
         else:
+            print 'something in the kategory is wrong!!! check !!!'
```

Comment:
Can this case happen? If the user is meant to see it, it should probably be something like 
```
else:
    print("Unknown fish category: %s" % category)
```

use () with print for python 3 compatibility throughout the whole code!